packages/terra-form-textarea/src/Textarea.module.scss

```diff
@@ -46,6 +51,11 @@
       resize: none;
     }
 
+    &[disabled]::placeholder {
+      color: var(--terra-form-textarea-placeholder-disabled-color, #70787c);
+      font-style: var(--terra-form-textarea-placeholder-disabled-font-style);
+    }
+
```

Comment:
The disabled placeholder styles are correct, but look incorrect since the other disabled values (background-color, border-color, color, opacity) also need to be updated, plus wouldn't hurt to also add no pointer-events:

```
    &[disabled] {      
      background-color: var(--terra-form-textarea-disabled-background-color, rgba(111, 116, 119, 0.2));
      border-color: var(--terra-form-textarea-disabled-border-color, #868a8c);
      border-style: var(--terra-form-textarea-disabled-border-style, solid);
      border-width: var(--terra-form-textarea-disabled-border-width, 1px);
      color: var(--terra-form-textarea-disabled-color, #1c1f21);
      cursor: default;
      opacity: var(--terra-form-textarea-disabled-opacity, 0.25);
      pointer-events: none;
      resize: none;
    }
```